lib/rubocop/cop/autocorrect_logic.rb

```diff
@@ -19,7 +19,7 @@ def support_autocorrect?
       def autocorrect_enabled?
         # allow turning off autocorrect on a cop by cop basis
         return true unless cop_config
-        cop_config['AutoCorrect'] != 'False'
+        !['False', false].include?(cop_config['AutoCorrect'])
```

Comment:
Hmm, seems I overlooked this originally. I think it practice it makes sense to support just the boolean value.